README.md

```diff
@@ -22,12 +22,16 @@ const vast = new VAST()
 
 Automatically generated API documentation can be found on
 [GitHub Pages](https://zentrick.github.io/iab-vast-model/). You can also
-open `docs/index.html` in a Web browser. Running `gulp doc` will (re)build the
+open `docs/index.html` in a Web browser. Running `npm run doc` will (re)build the
```

Comment:
Probably not something to discuss under this PR, but I wonder why we have the docs on master rather than a separate gh-pages branch. There's quite a bit of static stuff here.